src/routes/userTransform.ts

```diff
@@ -5,6 +5,8 @@ import { UserTransformController } from '../controllers/userTransform';
 
 const router = new Router();
 
+router.post('/workspaces/:wId/reconcileFunction', UserTransformController.reconcileFunction);
```

Comment:
Swagger doc

Reply:
It's not a public api and hence no need for swagger doc.